drupal-org.make

```diff
@@ -76,7 +76,6 @@ projects[views_infinite_scroll][type] = module
 projects[views_infinite_scroll][version] = 1.3
 projects[workbench_moderation][type] = module
 projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/2668006-2.patch
-projects[workbench_moderation][patch][] = https://www.drupal.org/files/issues/workbench_moderation-invalid_service_definition.patch
 projects[workbench_moderation][version] = 1.1
```

Comment:
Ah, I take it back -- it seems we were pegged to 1.1 because we were patching it (duh). Anyway -- as far as I can tell, this PR is only removing the patch, but is not actually updating WBM to 1.2. You'll need to change the constraint in composer.json as well, and run composer update to re-generate the make files.